benchmarks/profiler/utils/config_modifiers/parallelization_mapping.py

```
@@ -169,7 +168,9 @@ def _validate_intermediate_size(
 
 
 def get_candidate_parallel_mappings(
-    num_gpus: int, model_info: ModelInfo, phase: str
+    num_gpus: int,
+    model_info: ModelInfo,
+    phase: str,
```

Ruff (0.14.14)

[warning] Unused function argument: `phase`

(ARG001)